### PR TITLE
99714 client routing

### PR DESCRIPTION
**Pull request by numbap**
## [ADO-99714](https://dev.azure.com/VP-BD/DECD/_workitems/edit/99714)
Fixed [AB#99714](https://dev.azure.com/VP-BD/d5ca3cd4-19cb-4c8c-b3e6-a7068f4677e4/_workitems/edit/99714)

### Description

List of proposed changes:

- Used beta DS to add Link to menu
- Testing build. Please do not merge yet.

### CODE REVIEW

**Review by nibivi77 — COMMENTED** [review]
Seems to function properly. The only thing I noticed is when you navigate to another page, the drop-down menu stays open.

image

I suspect that this comes from Design System. Do we care about this?

**Review by DaceyTom2 — DISMISSED** [review]
Seems to work well. The first time I clicked on the menu it wasn't changing the page. Worth keeping a mental note to check from time to time

**Review by shewood — CHANGES_REQUESTED** [review]
to skip an entire test you just add .skip like this
it.skip('Validate da....

File: components/Layout.js [hunk 1]
```diff
@@ -25,7 +26,11 @@ export default function Layout(props) {
 
   return (
     <>
-      <MetaData language={props.locale} data={props.meta}></MetaData>
+      <MetaData
+        language={props.locale}
+        data={props.meta}
+        lang={!props.locale ? 'en' : props.locale}
```
Comment (nibivi77) [inline components/Layout.js:32]: I think `lang` prop can be removed since its not being used in <MetaData> and that `language` is doing the same thing.

**Review by nibivi77 — CHANGES_REQUESTED** [review]

**Review by nibivi77 — APPROVED** [review]
Awesome!

**Pull request closed without merging**